src/Compilers/Core/Portable/Symbols/Attributes/CommonEventWellKnownAttributeData.cs

```diff
@@ -27,5 +27,25 @@ public bool HasSpecialNameAttribute
             }
         }
         #endregion
+
+        #region ExcludeFromCodeCoverageAttribute
+
+        private bool _hasExcludeFromCodeCoverageAttribute;
```

Comment:
>private bool _hasExcludeFromCodeCoverageAttribute; [](start = 8, length = 50)

Do we really want to persist this information and spend time detecting these attributes during binding all the time?